README.md

```diff
@@ -27,13 +27,13 @@ Read more at [hasura.io](https://hasura.io) and the [docs](https://docs.hasura.i
 ## Features
 
 * **Make powerful queries**: Built-in filtering, pagination, pattern search, bulk insert, update, delete mutations
-* **Realtime**: Convert any GraphQL query to a live query by using subscriptions
+* **Real-time**: Convert any GraphQL query to a live query by using subscriptions
```

Comment:
@shashwatjain8 We are fine with the spelling and use it all over (see [website](https://hasura.io)).

Reply:
@dsandip Spellings are fine...Just need to merge the console help button link